spack-build-instructions/spack-developer-workflows.md

```diff
@@ -0,0 +1,257 @@
+# Spack workflows for developing Key4HEP software
+
+Using spack to develop software is somewhat pushing its intended usage to its limits.
+However, it is not impossible and this is an area of spack that is currently under active development.
+Unfortunately, this also means that the spack documentation might not be fully up-to-date on these topics.
```

Comment:
I think these sentences could be removed, as its clear that spack will be supporting this with all the dedicated commands.